go/cmd/dolt/commands/credcmds/check.go

```diff
@@ -82,27 +82,27 @@ func (cmd CheckCmd) Exec(ctx context.Context, commandStr string, args []string,
 	help, usage := cli.HelpAndUsagePrinters(cli.CommandDocsForCommandString(commandStr, checkDocs, ap))
 	apr := cli.ParseArgsOrDie(ap, args, help)
 
-	endpoint := loadEndpoint(dEnv, apr)
+	host, endpoint := loadEndpoint(dEnv, apr)
 
 	dc, verr := loadCred(dEnv, apr)
 	if verr != nil {
 		return commands.HandleVErrAndExitCode(verr, usage)
 	}
 
-	verr = checkCredAndPrintSuccess(ctx, dEnv, dc, endpoint)
+	verr = checkCredAndPrintSuccess(ctx, dEnv, dc, host, endpoint)
 
 	return commands.HandleVErrAndExitCode(verr, usage)
 }
 
-func loadEndpoint(dEnv *env.DoltEnv, apr *argparser.ArgParseResults) string {
+func loadEndpoint(dEnv *env.DoltEnv, apr *argparser.ArgParseResults) (string, string) {
 	earg, ok := apr.GetValue("endpoint")
 	if ok {
-		return earg
+		return "", earg // TODO
```

Comment:
Should I parse the endpoint here to get the host?

Reply:
Yeah, I think it's OK to do that in this case. We can also drop any error from the Parse and use a default audience if we want to...